.travis.yml

```diff
@@ -78,89 +50,91 @@ matrix:
         # all supported numpy versions.
         - os: osx
           stage: Cron tests
-          env: SETUP_CMD='test --remote-data=astropy'
-               CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang"
-               PIP_DEPENDENCIES="jplephem $ASDF_PIP_DEP"
-               CCOMPILER=clang
-               EVENT_TYPE='cron'
+          env: PYTHON_VERSION=3.6
+               TOXENV=py36-setup-all
+               TOXARGS='--remote-data=astropy'
+               # EVENT_TYPE='cron'
+               # CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang"
+               # CCOMPILER=clang
```

Comment:
I don't think it would work: I can pass CC to the tox env, but currently the CONDA_DEPENDENCIES are not installed in the tox-conda envs. There are still installed by ci-helpers in the root env though, which is useless. If we want to keep these customization of the test env by ci-helpers, we could split it in at least two functions: one to just install miniconda, and another that would be called in the tox env.

Reply:
Also why using conda's clang here ? osx is using clang by default isn't it ?

Reply:
Sorry I didn't reply to this sooner - I think the motivation for this might have been when we were considering adding some OpenMP functionality, but since we didn't do that in the end I think it's fine to drop this.